data/json/monsters/insect_spider.json

```diff
@@ -1857,5 +1857,347 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_aphid",
+    "type": "MONSTER",
+    "name": { "str": "giant aphid", "str_pl": "giant aphids" },
+    "description": "A green aphid about a size of a rabbit.  It is constantly searching for plants to suck from.  Probably you could sate it for a moment with cattle fodder.",
+    "default_faction": "ant",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "1600 ml",
+    "weight": "2080 g",
+    "hp": 20,
+    "speed": 120,
+    "material": [ "iflesh" ],
+    "symbol": "a",
+    "color": "light_green",
+    "morale": 20,
+    "aggression": -40,
+    "melee_skill": 1,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "melee_cut": 2,
+    "dodge": 3,
+    "armor_bash": 2,
+    "armor_cut": 5,
+    "armor_bullet": 3,
+    "vision_day": 10,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "fear_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT", "FIRE" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [ [ "EAT_FOOD", 60 ], [ "EAT_CROP", 80 ] ],
+    "reproduction": { "baby_monster": "mon_aphid", "baby_count": 1, "baby_timer": 20 },
+    "baby_flags": [ "SPRING", "SUMMER"],
+    "biosignature": { "biosig_item": "honeydew", "biosig_timer": 600 },
+    "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "PATH_AVOID_FIRE", "PATH_AVOID_FALL", "CATTLEFODDER", "PET_WONT_FOLLOW" ]
+  },
+  {
+    "id": "mon_mantis_small",
+    "type": "MONSTER",
+    "name": { "str": "huge praying mantis" },
```

Comment:
I suppose the name should be other, judging by its `id`?

Reply:
I've made ids and names based on other insects, such as wasps. Ex.:

"id": "mon_wasp_small" 
"name": { "str": "huge wasp" }

"id": "mon_wasp"
"name": { "str": "giant wasp" }

"id": "mon_wasp_mega"
"name": { "str": "whale wasp" }

"mon_mantis_small" is much larger than any other regular praying mantis, so I've decided to call it "huge", but relative to other mutated mantises it's "small"

Reply:
Ok, understandable.